recipes/servicestack/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "servicestack" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4874987b82e81a8eeb146b53df6b282f48f843aa1e4f17eecc24d723dc3c5247
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+  run:
+    - dataclasses >=0.6
+    - dataclasses-json >=0.5.4
+    - marshmallow >=3.12.2
+    - python >=3.9
+    - requests >=2.25.1
+
+test:
+  requires:
+    - pytest
+  commands:
+    - "${PREFIX}/bin/python3 -m unittest tests.test_serialization.TestSerialization"  # [linux]
```

I really would have expected this to work:
```suggestion
    - python -m unittest tests.test_serialization.TestSerialization
```
given [others in conda-forge](https://github.com/search?p=3&q=org%3Aconda-forge+%22-m+unittest%22&type=Code). Is there a reason you're only doing this on Windows? Also, is this actually using `pytest`, because it seems to only be using `unittest`.

---

I was having issues working out how to run these tests and found this approach in another conda-forge library so assumed it was needed but was likely for other reasons. This is nicer and does indeed select the right python version thanks! Removed `pytest` from requires.